data/clients.json

```diff
@@ -638,40 +634,47 @@
 		{
 			"name": "LEClient PHP library",
 			"url": "https://github.com/yourivw/LEClient",
-			"acme_v2": "true"
+			"acme_v2": "true",
+			"library": "PHP"
 		},
 		{
 			"name": "le-acme2-php library",
 			"url": "https://github.com/fbett/le-acme2-php",
-			"acme_v2": "true"
+			"acme_v2": "true",
+			"library": "PHP"
 		},
 		{
 			"name": "stonemax/acme2 PHP client",
 			"url": "https://github.com/stonemax/acme2",
-			"acme_v2": "true"
+			"acme_v2": "true",
+			"library": "PHP"
 		},
 		{
 			"name": "itr-acme-client PHP library",
 			"url": "https://github.com/ITronic/itr-acme-client",
-			"acme_v2": "true"
+			"acme_v2": "true",
+			"category": "PHP"
 		},
 		{
 			"name": "Certify The Web (Windows)",
 			"url": "https://certifytheweb.com",
-			"acme_v2": "v4 onwards"
+			"acme_v2": "v4 onwards",
+			"category": "Windows / IIS"
 		},
 		{
 			"name": "Ansible acme_certificate module",
 			"url": "https://docs.ansible.com/ansible/latest/modules/acme_certificate_module.html",
 			"acme_v2": "ansible >= 2.6",
 			"challenges": {
 				"TLS-ALPN-01": "ansible >= 2.7"
-			}
+			},
+			"category": "Python"
```

Comment:
```suggestion
			"category": "Ansible"
```
I'm not sure whether Ansible might not be better, since Ansible (while being based on Python) is more like its own ecosystem :)

Reply:
@jsha do you have an opinion? ("Ansible" needs to be added to the category list for that suggestion to be valid)

Reply:
Alternatively, it would fit into a category for configuration management tools / automation tools. That's probably more generic (and useful).

Reply:
I like the idea of a category for config management tools.